.github/workflows/test.yml

```diff
@@ -15,6 +15,3 @@ jobs:
 
       - name: Execute Tests
         run: make test-coverage
-
-      - name: Upload Coverage
```

Comment:
why did you delete it?

Reply:
Originally it was supposed to upload our code coverage to codecov and add a badge to the readme.
It never worked, because of some configurations that we needed to do in the organization level that we never did, so I removed it.